### ISSUE TITLE
bug in userbrowse.py

### ISSUE THREAD

**Issue by shmelsv** [Nicotine-Plus/nicotine-plus#2]
i m not coder and i m nubie with nicotine
but i found bug
it dont download some mp3s from some filelist
i removed part of code in userbrowse.py:

```
            if attrs[2]:
                bitrate += _(" (vbr)")
            try:
                rl = int(attrs[1])
            except ValueError:
                rl = 0
            length = "%i:%02i" % (int(rl // 60), rl % 60)
```

and now it works well

another bug (r its only for me)
totally black tabs as showed in pic. buddy list is ok. and when u click on tab it shows name of current tab. maybe becausse its not compiled? i dont know how to compile
http://fastpic.ru/view/66/2015/0320/2ad9184b22082b2c52eb6199ebbd41de.png.html

and i wanted ask to move download options like Download Directory and Download files from submenu to menu
is there is any chance that get new features? please. please. please
export filelist to xml (yeah i know that it can save filelist, but it would be beter to bz2.xml)
show locked share (to see what shared for buddylist only, as SoulseekQT has that)

**Comment by gfarmerfr** [comment]
Hi @shmelsv 

You should check if your problem still appear in the master branch.
This might have been fixed in this commit 925630378150cbcfe29464d1c6efab07760f3d05

**Comment by gfarmerfr** [comment]
Hi @shmelsv 
The second bug (black tabs) should have been fixed by this commit 5f8b33419e4d5d8694c659d64b637b7830416d11

**Comment by gfarmerfr** [comment]
Show locked share (to see what shared for buddylist only) is implemented on master.
